arch/arm/core/aarch64/arm_mmu.c

```diff
@@ -275,19 +274,17 @@ static void split_pte_block_desc(uint64_t *pte, int level)
 	set_pte_table_desc(pte, new_table, level);
 }
 
-/* Create/Populate translation table(s) for given region */
-static void init_xlat_tables(const struct arm_mmu_region *region)
+static void add_map(struct arm_mmu_ptables *ptables, const char *name,
```

Comment:
what's the purpose of the "name" parameter?
there's no expectation that the arch/ code needs to maintain an ontology of existing mappings, it can just blindly update the page tables.

Reply:
It's just for debugging purposes (see the `MMU_DEBUG` down here).